content/en/get-started/cards/vm-622.md

```diff
@@ -0,0 +1,13 @@
+### CDAP 6.2.2 VM
+
+#### Description
+
+  CDAP VM provides a complete, pre-built environment for running CDAP on your desktop or laptop.
```

Comment:
Should this be for VMs instead of "on your desktop or laptop"?

Reply:
So we ship the ova file (vm) that has CDAP pre-installed. So if users, say want to quickly try running CDAP on windows environment (desktop) they can just use the VM image and run it in their laptop.

Reply:
Thank you for the explanation!